bin/p2-dsctl/main.go

```diff
@@ -60,6 +64,7 @@ var (
 	updateMinHealth = cmdUpdate.Flag("minhealth", "The minimum health of the daemon set").String()
 	updateName      = cmdUpdate.Flag("name", "The cluster name (ie. staging, production)").String()
 	updatePodID     = cmdUpdate.Flag("pod", "The pod ID on the daemon set").String()
+	updateTimeout   = cmdUpdate.Flag("timeout", "Non-zero timeout for replicating hosts. e.g. 1m2s for 1 minute and 2 seconds").Default(TimeoutNotSpecified.String()).Duration()
```

Comment:
I believe that `Default()` will provide the behavior I need.
Placeholder doesn't seem to give me the default value of `TimeoutNotSpecified` for me.